tagInput.js

```diff
@@ -41,14 +50,24 @@ function addTags(){
     })
 
 }
-input.addEventListener('keyup',function(e){
+
+input.addEventListener('keypress',function(e){
     if(e.key === 'Enter' || e.key === " "){
-        tags.push(input.value);
-        addTags();
-        input.value='';  
+        if(input.value.trim() !== "") {
+            console.log("input value"+input.value.trim()+"....");
+            tags.push(input.value.trim());
+            addTags();
+            input.value='';
+        }
+    }
 
+    if(tags.length === tagLimit) {
```

Comment:
please maintain a constant habit for having constant names all in capital with _ as word separator.

Reply:
Done, please check now